test/integration/secrets/secrets_test.go

```diff
@@ -48,6 +51,56 @@ func TestSecrets(t *testing.T) {
 	defer framework.DeleteNamespaceOrDie(client, ns, t)
 
 	DoTestSecrets(t, client, ns)
+	DoTestSecretsEmptyValue(t, client, ns)
+}
+
+// DoTestSecretsEmptyValue Test whether apiserver will judge the secret data inconsistency
+// if the value of map in secret data is the empty string when patch secret
+func DoTestSecretsEmptyValue(t *testing.T, client clientset.Interface, ns *v1.Namespace) {
```

Comment:
Can you please name this to include the fact that `immutable` is a critical part of the test?
e.g. DoTestSecretsImmutableWithEmptyValue

Reply:
Okay, I've made the correction.